app/Models/Attendee.php

```diff
@@ -44,7 +45,7 @@ public static function boot()
         parent::boot();
 
         static::creating(function ($order) {
-            $order->private_reference_number = str_pad(random_int(0, pow(10, 9) - 1), 9, '0', STR_PAD_LEFT);
+            $order->private_reference_number = \Str::Random(10);
```

Comment:
I would suggest making this the same length as the database field for brevity sake. Also lessen the chance of collisions. 

```
\Str::Random(15); // The same length as the migration
```